--- origsrc/sushi-0.2.1/src/Makefile-js.am	2011-05-13 14:51:55.000000000 -0500
+++ src/sushi-0.2.1/src/Makefile-js.am	2011-12-11 02:40:52.930298600 -0600
@@ -26,15 +26,17 @@ dist_jsutil_DATA = \
     js/util/format.js
 
 jsutil_built_sources = \
-    $(srcdir)/js/util/features.js \
-    $(srcdir)/js/util/path.js
+    js/util/features.js \
+    js/util/path.js
 
 BUILT_SOURCES += $(jsutil_built_sources)
 
 js/util/path.js: Makefile js/util/path.js.in
+	$(AM_V_at)test -d js/util || $(mkdir_p) js/util
 	$(AM_V_GEN) $(do_subst) $(srcdir)/js/util/path.js.in > $@
 
 js/util/features.js: Makefile js/util/features.js.in
+	$(AM_V_at)test -d js/util || $(mkdir_p) js/util
 	$(AM_V_GEN) $(do_subst) $(srcdir)/js/util/features.js.in > $@
 
 nodist_jsutil_DATA = \
