--- origsrc/sushi-3.8.1/src/Makefile-js.am	2013-01-14 10:16:53.000000000 -0600
+++ src/sushi-3.8.1/src/Makefile-js.am	2013-08-08 02:10:32.351474300 -0500
@@ -26,11 +26,12 @@ dist_jsutil_DATA = \
     js/util/totemMimeTypes.js
 
 jsutil_built_sources = \
-    $(srcdir)/js/util/path.js
+    js/util/path.js
 
 BUILT_SOURCES += $(jsutil_built_sources)
 
 js/util/path.js: Makefile js/util/path.js.in
+	$(AM_V_at)test -d js/util || $(mkdir_p) js/util
 	$(AM_V_GEN) $(do_subst) $(srcdir)/js/util/path.js.in > $@
 
 nodist_jsutil_DATA = \
